lib/src/components/supa_email_auth.dart

```diff
@@ -95,6 +95,10 @@ class SupaEmailAuth extends StatefulWidget {
   /// Localization for the form
   final SupaEmailAuthLocalization localization;
 
+  /// Icons or custom prefix widgets for email UI
+  final Widget? prefixIconsEmail;
+  final Widget? prefixIconsPassword;
```

Comment:
Would it make sense to make it singular?
```suggestion
  final Widget? prefixIconEmail;
  final Widget? prefixIconPassword;
```